demux/demux_lavf.c

```diff
@@ -1121,12 +1121,12 @@ static int demux_open_lavf(demuxer_t *demuxer, enum demux_check check)
         t = av_dict_get(c->metadata, "title", NULL, 0);
         int index = demuxer_add_chapter(demuxer, t ? t->value : "",
                                         c->start * av_q2d(c->time_base), i);
-        mp_tags_copy_from_av_dictionary(demuxer->chapters[index].metadata, c->metadata);
+        mp_tags_move_from_av_dictionary(demuxer->chapters[index].metadata, &c->metadata);
```

Comment:
After looking into this a bit I don't think chapters can be updated in a way that is relevant for this bug, doesn't hurt to use the 'move' variant however